Cargo.toml

```diff
@@ -1,2 +1,6 @@
 [workspace]
 members = ["geo", "geo-types", "geo-postgis"]
+
+[patch.crates-io]
+geo = { path = "geo" }
+geo-types = { path = "geo-types" }
```

Comment:
What overrides does this effectively do? IIRC, `geo/Cargo.toml` already specifies `path = "../geo-types"`

Reply:
Good point, I guess this ended up not being  relevant. 

The context is that I was working on integrating the JTS test suite which relied on the WKT crate. WKT depends on geo-types, but because the WKT crate's Cargo.toml doesn't specify the local path to geo-types (nor should it) we were using the local geo-types for geo, but the remote geo-types for WKT, so different versions.

I ended up not including WKT (yet!) so I'll remove this for now, but I expect to revisit this soon.

> Would be great if we had tests for the new implementations, but no worries if you don't have capacity

In case you missed it, there are tests (in fact this work was driven by it), but I agree it's admittedly awkward that the tests are not including in this PR. 

My plan is, once https://github.com/michaelkirk/jts-test-runner is reviewed, to add it as a dev dependency and run the tests as part of geo, I originally planned to do this as a followup.

But I'd also be happy with this course of action:

- https://github.com/michaelkirk/jts-test-runner/pull/1 is reviewed
- jts-test-runner is moved to georust, and 0.1 published
- this pr is updated to run the centroid tests from jts-test-runner
- this pr is merged

Reply:
This all sgtm! Thanks for the explanation

Reply:
Now that I've integrated the jts-test-runner into this PR the patch lines are back.